boards/arm/blackpill_f401ce/blackpill_f401ce.dts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2020 Kalyan Sriram <coder.kalyan@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f401Xe.dtsi>
+#include <st/f4/stm32f411c(c-e)ux-pinctrl.dtsi>
```

Comment:
Thank you for your contribution 👍 
Please include the correct pinctrl definition. (even though there are not so many difference between f411ce and f401ce).
```suggestion
#include <st/f4/stm32f401c(d-e)ux-pinctrl.dtsi>
```

Reply:
@lochej Thanks for reminding me, I was going to mention that in my PR. Could you please explain the syntax on line 9? I don't understand what (if any) regex/replace is being performed and what file it is referencing. When I simply replaced f411 to f401 west was unable to find the right file. In your suggestion, you said change the c to a d, why is that?

Reply:
We recently moved from pinmux.c to device tree pinmux definition using the pinctrl device. So users need to get used to it 😄 

Those includes are automatically generated in the **hal_stm32** repository.
Here is the path where you can find all the pinctrl.dtsi:
```
zephyrproject/modules/hal/stm32/dts/st/f4/
```

Then find the dtsi which matches your SOC. I do it manually most of the time when porting or adding boards.